### PR TITLE
Reduce vulnerability interval

### PR DESCRIPTION
**Pull request by chiiph**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -49,12 +49,16 @@ services:
       FLEET_BETA_SOFTWARE_INVENTORY: 1
       FLEET_LICENSE_KEY: ${FLEET_LICENSE_KEY}
       FLEET_OSQUERY_LABEL_UPDATE_INTERVAL: 1m
+      FLEET_VULNERABILITIES_CURRENT_INSTANCE_CHECKS: "yes"
+      FLEET_VULNERABILITIES_DATABASES_PATH: /vulndb
+      FLEET_VULNERABILITIES_PERIODICITY: 20m
```
Comment (chiiph) [inline docker-compose.yml]: Note that this PR really only changes this. The rest is bringing master up to date with production.

**Review by lucasmrod — APPROVED** [review]

**Comment by mna** [comment]
@chiiph Note that this should be merged to `production` branch too, as this is the one used (or to be used) by 4.5+.

**Comment by mna** [comment]
Ah nevermind, I see this was done!

**Comment by mna** [comment]
Ah nevermind, I see this was done!

**Pull request merged**